rubix-hadoop2/pom.xml

```diff
@@ -17,10 +17,17 @@
 
         <dependency>
             <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-core</artifactId>
+            <artifactId>hadoop-common</artifactId>
+            <version>${dep.hadoop2.version}</version>
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-hdfs</artifactId>
```

Comment:
Where are we using this dependency. Do we need this ?

Reply:
Due to the `hadoop-core` -> `hadoop-common` dependency update, this is required to provide the `DistributedFileSystem` class for `CachingDistributedFileSystem`.

Reply:
sounds good.

Reply:
could the classes from this clash with the installed one? hadoop-common is provided scope while this will be packed in assembly due to compile time dependency.

We need integration tests that can smoke test PRs against presto and hive to answer such questions. Probably something to pick up soon

Reply:
@stagraqubole I agree that integration tests should be an upcoming priority for testing situations like this.